### ISSUE TITLE
Installation fails on MacOS Catalina

### ISSUE THREAD

**Issue by philhinzatucsc** [sblunt/orbitize#242]
This is likely a newbie issue, but I am unable to install orbitize (and radvel?) on macOS Catalina.  I am following the code astro tutorials and started by trying to follow the Day0 instructions there to install orbitize:

https://github.com/semaphoreP/codeastro/blob/main/Day0/INSTALL.md

If I run "pip install orbitize" it appears to fail while trying to compile C code related to radvel with the relevant error (I think) being:

 fatal error: too many errors emitted, stopping now [-ferror-limit=]
  1 warning and 20 errors generated.
  error: command '/usr/bin/clang' failed with exit code 1

Any pointers on what I am missing?  I assume installation on Catalina is a common thing.  Some comments:
1) I am using anaconda and followed the day0 instruction to create a codeastro python environment
2) I noticed, under "Installation-Issues?" the suggestion to try "pip install orbitize --install-option="--disable-cython"  ".  When I do so I get "error: option --disable-cython not recognized"
3) I noticed the X-code command line tools requirement. I manually uninstalled and reinstalled the Xcode command line tools.  This did not help.

I included the raw output from "pip install orbitize" command in the attachment.

Phil
[orbitize_errors.txt](https://github.com/sblunt/orbitize/files/6773643/orbitize_errors.txt)

Phil

**Comment by semaphoreP** [comment]
Hi Phil,

I'm not a mac user but we recently ran into a similar issue with another user in Code/Astro. The fix was to run `brew upgrade llvm` before trying to install orbitize/radvel now. Let me know if this helps! Otherwise, perhaps a mac user can help you out.

**Comment by philhinzatucsc** [comment]
Thanks for the suggestion.  As it turns out I didn't have llvm installed.  I ran "brew install llvm" and confirmed that llvm 12.0.0_1 is now installed.  However, I get the same error messages when running "pip install orbitize"

**Comment by sblunt** [comment]
Hey Phil,

Thanks for the bug report, and for your patience waiting for us to follow up. I'd first check that there are no outstanding issues with your gcc and cython installations. Make sure cython is up to date (`pip install cython --upgrade`) and try to compile a simple C program (try "hello world"-- I'm sure you can find a simple script like this online if needed). If all that's working, but your orbitize and radvel installations are still failing, let us know.

**Comment by philhinzatucsc** [comment]
Okay, I ran "pip install cython --upgrade". It installed fine (seemed to upgrade from 0.29.23 to 0.29.24).  I then copied a hello world program, compiled it via gcc and ran it.  It worked as expected.  I then tried "pip install orbitize" and saw the same errors.  Thanks for the suggestion, but it appears that is not gcc.

**Comment by philhinzatucsc** [comment]
I am reviewing what I did to get to this place.  First, I note that, following the day0 instructions I actually cloned the git repository and ran  "pytest --mode codeastro" which fails by saying "ImportError while importing test module '/Users/phinz/codeastro/orbitize/tests/test_radvel_utils.py'" So, that's when I tried running "pip install radvel" and "pip install orbitize".  Both seem to fail with very similar errors, related to some C compiler problem.  The errors all look like:

"error: implicit declaration of function 'assert' is invalid in C99 [-Werror,-Wimplicit-function-declaration]"  

so it seems I need to somehow change the C compiler that the installation package is using or the -W settings it is using?  I'm just guessing.

**Comment by philhinzatucsc** [comment]
So   . . . I did need a different C compiler, and Jasons's comment above was the correct clue.  I just wasn't careful to check that the install script was using the correct compiler after installing it.  The default on my system was to use /usr/bin/clang.  After running "brew upgrade llvm" I now had a different clang executable, but the output warns me it was not installed as the default. To use it I also needed to add:

echo 'export PATH="/usr/local/opt/llvm/bin:$PATH"' >> /Users/phinz/.bash_profile

so that it used the one just installed.  Now it works, and I even got a secret code!

Thanks for your feedback, Jason and Sarah.
